scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/types/package.scala

```diff
@@ -63,10 +65,15 @@ package object types {
    */
   case class Json(wkt: String)
   object Json {
+    // Use same mapper as the TableRowJsonCoder
     private lazy val mapper = new ObjectMapper()
+      .registerModule(new JavaTimeModule())
+      .registerModule(new JodaModule())
+      .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
```

Comment:
what's this needed for?

Reply:
this is the config from [TableRowJsonCoder](https://github.com/apache/beam/blob/d7c56914f80c6d0b817a3a69b58de0742ef8d235/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowJsonCoder.java#L80).

It forces serialization of dates as string
> Feature that determines whether Date (and date/ time) values (and Date-based things like java. util. Calendars) are to be serialized as numeric time stamps (true; the default), or as something else (usually textual representation).